src/App.js

```diff
@@ -1,24 +1,31 @@
 import React from 'react';
+import styled from 'styled-components';
 import logo from './logo.svg';
 import './App.css';
+import GlobalFonts from './fonts/fonts.js';
```

Comment:
@tariqporter how do we fix it so that i can just do `import GlobalFonts from 'fonts' instead of having to use the whole relative path? 😓

Reply:
aha that has to be added in webpack